lib/webpack-plugin.d.ts

```diff
@@ -0,0 +1,18 @@
+import { Compiler } from 'webpack';
```

Comment:
Please revert this file as it shouldn't be committed.

Reply:
fixed